send2cgeo.user.js

```diff
@@ -4,19 +4,8 @@
 // @description    Add button "Send to c:geo" to geocaching.com
 // @author         c:geo team and contributors
 // @require        http://code.jquery.com/jquery-3.4.1.min.js
-// @include        https://www.geocaching.com/play/search*
-// @include        https://www.geocaching.com/play/search/*
-// @include        http://www.geocaching.com/seek/cache_details*
-// @include        https://www.geocaching.com/seek/cache_details*
-// @include        https://www.geocaching.com/map/*
-// @include        https://www.geocaching.com/play/map*
-// @include        https://www.geocaching.com/play/map/*
-// @include        http://www.geocaching.com/geocache/*
-// @include        https://www.geocaching.com/geocache/*
-// @include        http://www.geocaching.com/my/recentlyviewedcaches*
-// @include        https://www.geocaching.com/my/recentlyviewedcaches*
-// @include        http://www.geocaching.com/seek/nearest*
-// @include        https://www.geocaching.com/seek/nearest*
+// @include        http*://www.geocaching.com/*
+// @exclude        /^https?://www\.geocaching\.com/(login|jobs|careers|brandedpromotions|promotions|blog|help|seek/sendtogps|profile|account)/
```

Comment:
There is also the /track/* tree for trackables. This should also be integrated in the @exclude's.
I don't know all subtrees which are in use by geocaching.com.
But we know on which sites the script should run.
Therefore I suggest to use only @include's instead of "uncounted" @exclude's.

Reply:
Should I do it as before, or in one row.
e.g. `/^https?://www\.geocaching\.com/(play/(search|map)|seek/(cache_details|nearest)|map|geocache|my/recentlyviewedcaches)/`

Reply:
Maybe a mix of both (for better readability), eg.
~~~
/^https?://www\.geocaching\.com/play/(search/|map/)/
/^https?://www\.geocaching\.com/seek/(cache_details\.|nearest\.|)/
/^https?://www\.geocaching\.com/my/recentlyviewedcaches\./
/^https?://www\.geocaching\.com/(map/|geocache/)/
~~~
Please end with / or \. to distinguish between folder and site.

Reply:
Yes, that's the best. search and map (the searchmap) can not end with a / because this sites use ?. At the searchmap is both possible (with and without /)